data_steward/cleaners/null_invalid_foreign_keys.py

```diff
@@ -0,0 +1,116 @@
+"""
+
+ Cleaning Rule 1 : Foreign key references (i.e. visit_occurrence_id in the condition table) should be valid.
+ (Existing Achilles rule - validating for foreign keys include provider_id, care_site_id, location_id, person_id,
+ visit_occurrence_id)
+
+
+"""
+
+# Project Imports
+import resources
+from constants.cleaners.clean_cdr import PERSON_TABLE_NAME
+
+FOREIGN_KEYS_FIELDS = ['person_id', 'visit_occurrence_id', 'location_id', 'care_site_id', 'provider_id']
+
+INVALID_FOREIGN_KEY_QUERY = '''
+            SELECT {cols} 
+            FROM {project}.{dataset_id}.{table_name} t 
+            {join_expr}
+            '''
+
+
+def mapping_table_for(domain_table):
```

Comment:
It seems like you are only using this function in this module.  You could give it python's version of a "protected" status by prefixing the name with an underscore, `_mapping_table_for`.  Then it will be available to your module, but will never be imported if someone were to use a wildcard import.  Single and double leading underscores are a pep 8 convention.  This [page](https://dbader.org/blog/meaning-of-underscores-in-python) does a nice job discussing this.

Reply:
Done